scout/constants/gene_tags.py

```diff
@@ -15,6 +15,20 @@
     ("other", "other - Other"),
 )
 
+# Associates an inheritance model with a color, for displaying inheritance on Variant page
+INHERITANCE_PALETTE = {
+    "AD": {"bgcolor": "bg-red", "text_color": "text-white"},
```

Comment:
Much nicer to my eyes! And one could still add an absolute color in the css if the var(bs-) ones are not good enough in the end.